src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,32 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
 
+        List<Manufacturer> manufacturerList = List.of(
+                new Manufacturer("Toyota", "Japan"),
+                new Manufacturer("Buick", "USA"),
+                new Manufacturer("Aston Martin", "UK"),
+                new Manufacturer("Tesla", "USA"),
+                new Manufacturer("Porsche", "Germany"),
+                new Manufacturer("Bugatti", "France"),
+                new Manufacturer("Pagani Automobili", "Italy"),
+                new Manufacturer("Seat", "Spain")
+        );
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        Manufacturer manufacturerToAdd = new Manufacturer(40L, "Maserati", "Italy");
```

Comment:
```suggestion
        Manufacturer manufacturerToAdd = new Manufacturer("Maserati", "Italy");
```
id should be set by dao method